src/pages/styles.less

```diff
@@ -83,18 +83,17 @@
   @apply text-green-200;
 }
 
-
 .error {
   @apply text-red-600;
 }
 
 .jsonPretty {
-  height: 600px;
+  height: 500px;
   max-height: 600px;
   max-width: 95vw;
   overflow: scroll;
   font-size: 1em;
-  background-color: rgb(249, 249, 249);
+  color: white;
```

Comment:
Let's use the tailwind `@apply` directive if possible

Reply:
Yeah...i have fixed it.